### PR TITLE
Added first boot script specific to GCE codelab image.

### PR DESCRIPTION
**Pull request by jtk54**
Added first boot script specific to GCE codelab image. It performs the config difference between what we want in the general case for GCE and the configuration specific for the codelab. @duftler @ewiseblatt PTAL.

### CODE REVIEW

File: install/first_gce_codelab_boot.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+if [[ `/usr/bin/id -u` -ne 0 ]];then
+  echo "$0 must be executed with root permissions; exiting"
+  exit 1
+fi
+
+/opt/spinnaker/install/first_google_boot.sh
+
+echo "Setting hostname for rosco deb repo"
+sed -i "s/debianRepository:.*/debianRepository: http:\/\/$(hostname):9999\/ trusty main/" /opt/rosco/config/rosco.yml
```
Comment (ewiseblatt) [inline install/first_gce_codelab_boot.sh]: Does this need to be tied to "trusty"? Shouldnt it be whatever the host is running?

File: install/first_gce_codelab_boot.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+/opt/spinnaker/bin/reconfigure_spinnaker.sh
```
Comment (ewiseblatt) [inline install/first_gce_codelab_boot.sh]: Do you need to restart spinnaker, or just rosco?

Comment (jtk54) [inline install/first_gce_codelab_boot.sh]: Most likely just rosco, good point.

**Comment by jtk54** [comment]
This startup script doesn't belong with the distribution -- I'll move it to another location in this repo so that it doesn't ship with the distribution, and modify the GCE codelab accordingly.

**Pull request closed without merging**